tasks.rb

```diff
@@ -99,6 +100,7 @@ def create_hpxmls
     'base-dhw-solar-thermosyphon-evacuated-tube.xml' => 'base.xml',
     'base-dhw-solar-thermosyphon-flat-plate.xml' => 'base.xml',
     'base-dhw-solar-thermosyphon-ics.xml' => 'base.xml',
+    'base-dhw-solar-direct-flat-plate-defaults.xml' => 'base-dhw-solar-direct-flat-plate.xml',
```

Comment:
Can you remove this file and incorporate the solar system into base-misc-defaults.xml?

Reply:
Actually, I don't see any references to base-dhw-solar-direct-flat-plate-defaults.xml, so it seems like you are already incorporating it in base-misc-defaults.xml....

Reply:
Oops.  I incorporated the solar system into base-misc-defaults.xml and forgot to remove the base-dhw-solar-direct-flat-plate-defaults.xml.